examples/gatsby-site/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "gatsby-starter-hello-world",
+  "private": true,
+  "description": "A simplified bare-bones starter for Gatsby",
+  "version": "0.1.0",
+  "license": "MIT",
+  "scripts": {
+    "build": "gatsby build",
+    "develop": "gatsby develop",
+    "format": "prettier --write \"**/*.{js,jsx,json,md}\"",
+    "start": "npm run develop",
+    "serve": "gatsby serve",
+    "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing \""
+  },
+  "dependencies": {
+    "gatsby": "^2.15.34",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "@hpe/project-scripts": "*",
+    "@hpe/react-hooks": "*"
+  },
+  "devDependencies": {
+    "prettier": "^1.18.2"
```

Comment:
I think `prettier` should not be a devDependency, or at least this definition is not consistent with the other packages dependencies.
If you strongly think it should be used as devDependency, please consider moving it up to the root `package.json` to act consistently with the other packages.

Reply:
`@hpe/project-scripts` will add prettier to the project. Let's remove it from the `package.json`.